client/galaxy/style/less/base.less

```diff
@@ -671,6 +671,10 @@ div.form-row-error-message {
     margin-top: 5px;
 }
 
+.workflow-right .right-content .section-row {
```

Comment:
@guerler I actually didn't want to change this across the board, only in this specific location.  It only looks really clumpy to me in the right pane of the workflow, hence the other class.